internal/Picker/PickerItem.js

```diff
@@ -0,0 +1,30 @@
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import Marquee from '@enact/ui/Marquee';
+
+import css from './Picker.module.less';
+
+const PickerItemBase = kind({
+	name: 'PickerItem',
+
+	styles: {
+		css,
+		className: 'item',
+		publicClassNames: true
+	},
+
+	computed: {
+		className: ({children, styler}) => styler.append({numeric: !isNaN(Number(children))})
```

Comment:
`Number. isNaN` might be better instead of `isNan` to check if `Number(children)` is exactly `NaN` or not.

Reply:
We actually don't need this computed code now.